infra/terraform/test-org/org/external_data/foo.csv

```diff
@@ -1,3 +1,3 @@
 id,name,dept
 1,Keith,TP
-2,Madeline,HR
\ No newline at end of file
+2,Madeline,HR

```

Comment:
Likely unimportant, but this will force a replace of `google_storage_bucket_object.ci_bq_external_hive_file_foo`